client/driver/docker.go

```diff
@@ -339,46 +343,57 @@ func (d *DockerDriver) Abilities() DriverAbilities {
 	}
 }
 
-func (d *DockerDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, error) {
+func (d *DockerDriver) Prestart(ctx *ExecContext, task *structs.Task) error {
```

Comment:
Can't believe I missed that in this PR! Added (inside the createImage method so it's not emitted if the image already exists).